Source/Extensions/Blazorise.Captcha/Captcha.razor.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Captcha;
+
+public partial class Captcha : BaseComponent
+{
+    /// <summary>
+    /// The current state of the Captcha.
+    /// </summary>
+    public CaptchaState State { get; private set; } = new();
+
+    /// <summary>
+    /// A Captcha solved event. 
+    /// Provides contextual information about the Captcha state after the user has solved.
+    /// </summary>
+    [Parameter] public EventCallback<CaptchaState> OnSolved { get; set; }
```

Comment:
We usually don't have `On` prefixes. Just name them in past tense, `Validated`, `Expired`.

Reply:
`Validated`? Maybe `Validating`? `Validated` makes me think it's done validating. And the purpose is for the user to apply validation. Anyway refer to : 
> We have a `OnValidate` event which we could ommit, and the user would do any validations on the OnSolved event instead? What do you think?

Maybe we should just remove the validate event, and if the user needs to do validation he does it on the `Solved` event?

Reply:
If it's an ongoing validation, then I would name it `Validate`.